doc/api/process.md

```diff
@@ -2601,6 +2601,77 @@ the [`'warning'` event][process_warning] and the
 [`emitWarning()` method][process_emit_warning] for more information about this
 flag's behavior.
 
+## `process.permission`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+* {Object}
+
+This API is available through the [`--experimental-permission`][] flag.
+
+`process.permission` is an object whose methods are used to manage permissions
+for the current process. Additional documentation is available in the
+[Permission Model][].
+
+### `process.permission.check(scope[, reference])`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+* `scopes` {string}
+* `reference` {string}
+* Returns: {boolean}
+
+Verifies that the process is able to access the given scope and reference.
+If no reference is provided, a global scope is assumed.
+
+The reference has a meaning based on the provided resource. For example,
+the reference when the scope is File System means files and folders.
+
+The available scopes are:
+
+* `fs` - All File System
+* `fs.read` - File System read operations
+* `fs.write` - File System write operations
```

Comment:
I'm curious why we need to have multiple names for seemingly the same thing. For example, why do we need something called `FileSystemIn`, but also `fs.read`? Could we reuse names to reduce confusion?

Reply:
Actually, I'll adjust it. In the first implementation, I was using FileSystemOut and FileSystemIn for write and read respectively. I'll adjust it in the next commit.